src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -54,4 +56,12 @@ public void statusCodeIs(int arg0) {
         final int expectedStatus = 204;
         Assert.assertEquals(APIRequestManager.getStatusCode(), expectedStatus);
     }
+
+    @When("^I post a new story to \"([^\"]*)\"$")
+    public void iPostANewStoryTo(final String endpoint, final Map<String, String> values)  {
+
+        String newEndpoint = APICommons.buildEndPoint(endpoint);//APIRequestManager.getResponse());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)